### PR TITLE
chore: add Windows Server 2k25 template

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
chore: add Windows Server 2k25 template

The OS has medium and large flavor, server and highperformance workload.

**Release note**:
```
chore: add Windows Server 2k25 template
```

### CODE REVIEW

File: templates/windows2k25.tpl.yaml [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+            name: ${NAME}
+          name: rootdisk
+        networks:
+        - name: default
+          pod: {}
+parameters:
+- name: NAME
+  description: VM name
+  generate: expression
+  from: "windows2k25-[a-z0-9]{6}"
```
Comment (0xFelix) [inline templates/windows2k25.tpl.yaml]:
```suggestion
  from: "windows2025-[a-z0-9]{6}"
```

File: templates/windows2k25.tpl.yaml [hunk 2]
```diff
@@ -0,0 +1,204 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: windows2k25-{{ item.workload }}-{{ item.flavor }}
+  annotations:
+    openshift.io/display-name: "Microsoft Windows 2k25 VM"
```
Comment (0xFelix) [inline templates/windows2k25.tpl.yaml:6]:
```suggestion
    openshift.io/display-name: "Microsoft Windows Server 2025 VM"
```

Comment (ksimon1) [reply templates/windows2k25.tpl.yaml]: fixed

File: templates/windows2k25.tpl.yaml (hunk 1 above)
Comment (ksimon1) [reply templates/windows2k25.tpl.yaml]: fixed

**Review by 0xFelix — APPROVED** [review]
/approve
/lgtm

**Comment by ksimon1** [comment]
Merging, because we don't have e2e test for it yet

**Pull request merged**